addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.xbmc.versioncheck"
     name="Version Check"
-    version="0.3.281"
+    version="0.3.30"
```

Comment:
I'm wondering if there's a reason for the high patch number we had here before

Reply:
I proposed 4 commits and in each I propose to add +0.1
but I can suggest a 0.3.27